services/121-service/src/registration/validators/registrations-input-validator.ts

```diff
@@ -402,9 +402,12 @@ export class RegistrationsInputValidator {
   private createLanguageMapping(
     programLanguages: string[],
   ): Record<string, string> {
-    const languageNamesApi = new Intl.DisplayNames(['en'], {
-      type: 'language',
-    });
+    const languageNamesApi = new Intl.DisplayNames(
+      [RegistrationPreferredLanguage.en],
+      {
+        type: 'language',
+      },
+    );
```

Comment:
it seems off here now, that we use this api here, whereas elsewhere we now use fully defined lists of languages and list of language linguonyms defined in our repo. Could those not also use this API? Or vice versa, could this use a list like that as well?